src/etc/inc/interfaces.inc

```diff
@@ -2590,6 +2590,8 @@ function interface_track6_configure($interface = 'lan', $lancfg, $linkupevent =
                  */
                 $parentrealif = get_real_interface($lancfg['track6-interface']);
                 killbypid("/var/run/dhcp6c_{$parentrealif}.pid", 'HUP');
+            }  else {
+                services_dhcpd6_configure_only(array(),true);
```

Comment:
why not services_dhcpd_configure('inet6') ?

Reply:
Because I did not want to affect anything else by calling all the other stuff in that call, just the two radvd and dhcpd6

Reply:
inet(4) is skipped, but you need the chroot fixup if dhcp wasn't started already

Reply:
OK, if you think it better... you da boss.